sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/sas/TableSasProtocol.java

```diff
@@ -1,13 +1,16 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-package com.azure.data.tables.implementation;
+
+package com.azure.data.tables.sas;
+
+import com.azure.data.tables.implementation.StorageConstants;
 
 import java.util.Locale;
 
 /**
  * Specifies the set of possible permissions for Shared Access Signature protocol.
  */
-public enum SasProtocol {
+public enum TableSasProtocol {
```

Comment:
Any reason we aren't using `SasProtocol` from `azure-storage-common`?

Reply:
I think `azure-storage-common` dependency was removed from `azure-data-tables`. So, types defined in storage common cannot be used here.

Reply:
We removed the `azure-storage-common` dependency and copied over a lot of classes from there to handle parsing connection strings and this, for example.